src/bitriseAPI/triggerStart.js

```diff
@@ -29,57 +30,66 @@ const triggerStart = async (initialWorkflow='') => {
             }
         }
         while (!SKIP_TESTS) {
-                SKIP_TESTS = readline.question(`Skip tests? (y/n):\n`, {
-                    limit: YES_NO_OPTIONS,
-                    limitMessage: `Type y or n!`
-                });
-            }
+            SKIP_TESTS = readline.question(`Skip tests? (y/n):\n`, {
+                limit: YES_NO_OPTIONS,
+                limitMessage: `Type y or n!`
+            });
+        }
 
 
         const skippingTestsText = YES_OPTIONS.includes(SKIP_TESTS) ? 'Tests will be skipped' : 'Tests will run'
         console.log(consoleBlue, `Triggering ${WORKFLOW} for branch ${BRANCH}. ${skippingTestsText}`);
     }
 
     //Create payload
-    const BITRISE_BUILDS_URL = 'https://api.bitrise.io/v0.1/apps/af50b4926a122ad0/builds';
-    const payload = {
-        "hook_info": {
-            "type": "bitrise"
+    const envVariables = [
+        {
+            "is_expand": true,
+            "mapped_to": "SKIP_TESTS",
+            "value": YES_OPTIONS.includes(SKIP_TESTS) ? "true" : "false"
         },
-        "build_params": {
-            "branch": `${BRANCH}`,
-            "workflow_id": `${WORKFLOW}`,
-            "base_repository_url": "https://github.com/camelotls/ie-native-app",
-            "environments": [
-                {
-                    "is_expand": true,
-                    "mapped_to": "SKIP_TESTS",
-                    "value": YES_OPTIONS.includes(SKIP_TESTS) ? "true" : "false"
-                },
-            ]
-        }
-    }
-    let response = await triggerBuild(BITRISE_BUILDS_URL, payload);
-    if (response.statusCode > 201) {
+    ]
+    const payload = createPayload(BRANCH, WORKFLOW, envVariables);
+    // const payload = {
+    //     "hook_info": {
+    //         "type": "bitrise"
+    //     },
+    //     "build_params": {
+    //         "branch": `${BRANCH}`,
+    //         "workflow_id": `${WORKFLOW}`,
+    //         "base_repository_url": "https://github.com/camelotls/ie-native-app",
+    //         "environments": [
+    //             {
+    //                 "is_expand": true,
+    //                 "mapped_to": "SKIP_TESTS",
+    //                 "value": YES_OPTIONS.includes(SKIP_TESTS) ? "true" : "false"
+    //             },
+    //         ]
+    //     }
+    // }
```

Comment:
If these are not needed, could you please remove them?

Reply:
I might do that, but shouldn't would stash code for when the difficult times come?